src/languages/es.ts

```diff
@@ -515,6 +515,7 @@ const translations = {
         validate: 'Validar',
         expenseReports: 'Informes de Gastos',
         rateOutOfPolicy: 'Tasa fuera de póliza',
+        editYourProfile: 'Edita tu perfil',
```

Comment:
@twilight2294 Please add the discussion here where you confirmed this Spanish translation.